README.md

```diff
@@ -6,7 +8,8 @@
 - `Yarn (1.x or later)` [download site](https://yarnpkg.com/getting-started/install)
 - `.env` **Please ask project owner privately for the environment variables.**
 
-## Installation Steps
+## Run locally
+*** Now that the developers are quite busy, the `development`and the `master` branch are not stable, please use the `pre-development` branch.
```

Comment:
```suggestion
***Now that the developers are quite busy, the `development` and the `master` branch are not stable, please use the `pre-development` branch.***
```
Just a small fix here 🧐

Reply:
Oh, no. I did it intentionally 😂  Like a disclaimer thing